tuist/tuist
# Re-enable swiftformat

- Removing empty test file (caused swiftformat to error -  see https://github.com/nicklockwood/SwiftFormat/issues/446) 
- Doing a sweep of swiftformat fixes

Test Plan:

- Verify tuist builds via `swift build`
- Verify tests pass via `swift tests`
- Verify swiftformat runs without issues
- Verify swiftformat runs on CI
